docs/reference/aggregations/pipeline/bucket-sort-aggregation.asciidoc

```diff
@@ -0,0 +1,126 @@
+[[search-aggregations-pipeline-bucket-sort-aggregation]]
+=== Bucket Sort Aggregation
+
+A parent pipeline aggregation which sorts the buckets of its parent multi-bucket aggregation.
+Zero or more sort fields may be specified together with the corresponding sort order.
+Each bucket may be sorted based on its `_key`, `_count` or its sub-aggregations.
+In addition, parameters `from` and `size` may be set in order to truncate the result buckets.
+
+NOTE: The `bucket_sort` aggregation, like all pipeline aggregations, is executed after all other sibling aggregations.
```

Comment:
"after all other sibling aggs" <-- potentially confusing since there are sibling pipeline aggs.  

Maybe something like *"... after all other bucket and metric aggregations"*?  Or *"... after all non-pipeline aggregations"*?